data/lib/others/reward_boss_lib.lua

```diff
@@ -60,6 +60,20 @@ function MonsterType.createLootItem(self, lootBlock, chance, lootTable)
 		end
 	end
 
+	local itemType = ItemType(lootBlock.itemId)
+	local decayTo = itemType:getDecayId()
+	local decayTime = itemType:getDecayTime()
+	if decayTo and decayTo >= 0 and decayTime and decayTime ~= 0 then
+		local transformDeEquipId = itemType:getTransformDeEquipId()
+		if transformDeEquipId and transformDeEquipId > 0 then
+			print("[Warning - MonsterType.createLootItem] Convert boss '" .. self:name() .. "' reward ID '" .. lootBlock.itemId .. "' to ID " .. transformDeEquipId .. ".")
```

Comment:
You can changue here to 
```suggestion
			Spdlog.warn("[MonsterType.createLootItem] - Convert boss '" .. self:name() .. "' reward ID '" .. lootBlock.itemId .. "' to ID " .. transformDeEquipId .. ".")
```